lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -69,7 +69,7 @@ svgwrite = "*"
 pyparsing = "*"
 "Fabric3" = "*"
 paramiko = "*"
-python-genomespaceclient = "<1.0"
+python-genomespaceclient = "<2.0"
```

Comment:
why this is not `1.0.0`?

Reply:
To allow minor release upgrades, e.g. the already released 1.1.0 . Is there a problem?

Reply:
I might be missing something here; is not this preventing upgrading cloudbridge version to v1 in #6660?